recipe/meta.yaml

```diff
@@ -59,3 +59,5 @@ about:
 extra:
   recipe-maintainers:
     - xylar
+    - tomvothecoder
+    - chengzhuzhang
```

Comment:
@tomvothecoder and @chengzhuzhang, please make sure I got these right.

Reply:
Yes, it is correct. Thank you for taking care of this!